codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -176,15 +175,72 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         }
     }
 
+    @Override
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
```

Comment:
To make sure I understand, these have been overridden to show/guarantee they don't throw an exception? Could that be documented here and for flush so they aren't changed without realizing they shouldn't throw? Casual readers would delete them because they don't serve a functional purpose.